app/components/bookings/booking/booking.component.html

```diff
@@ -1,10 +1,47 @@
 <ion-card>
-    <ion-card-content>
-        <ion-grid>
-            <ion-row>
-                <ion-col width-20>{{ booking.effort | workdaysToHours }}</ion-col>
-                <ion-col width-80>{{ booking.description }}</ion-col>
-            </ion-row>
-        </ion-grid>
-    </ion-card-content>
+  <ion-card-content>
+    <ion-grid>
+      <ion-row *ngIf="withWpName" style="font-size: medium">
```

Comment:
Please do not use inline styles!!
This makes for html which is really hard to maintain and leads to a lot of repetition. It is much cleaner to assign classes and style them in the scss files. The html file should only define how elements are structured, while the scss file defines what the elements should look like.